__tests__/Welcome.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Welcome from '../src/components/Welcome';
+
+describe('Render Welcome component', () => {
+    it('should render Welcome component successfully', () => {
+        const wrapper = shallow(<Welcome/>);
+        expect(wrapper).toHaveLength(1);
```

Comment:
This isn't a proper test, the page needs to be tested properly e.g `expect(wrapper.find('h1').length).toBe(2);` and so on instead of just `expect(wrapper).toHaveLength(1);`